src/Enforce/chainables/compound/index.js

```diff
@@ -10,14 +10,10 @@ import { runtimeError } from 'Helpers';
  * @param {object} tests
  * @return {object} enforce object
  */
-function compound(value: AnyValue, group: string, tests: CompoundTestObject, allRules: Rules = {}): EnforceSelf {
-    if (this.valid === false) {
-        return this;
-    }
-
-    this.valid = runners[group](value, tests, allRules);
+function compound(value: AnyValue, group: string, tests: CompoundTestObject, allRules: EnforceRules = {}): EnforceProxy {
```

Comment:
Not a technical reason.
I want to prevent people from needlessly using `Any`, as it is considered a `weak` type. If they use a type, they need to explicitly state it. This is why I added an `eslint` rule against using `Any`. `AnyValue` is my way of making `Any` explicit for API functions.